cdap-master/src/main/java/co/cask/cdap/master/environment/k8s/AbstractServiceMain.java

```diff
@@ -164,9 +154,6 @@ protected void configure() {
           .toProvider(new SupplierProviderBridge<>(masterEnv.getDiscoveryServiceSupplier()));
         bind(DiscoveryServiceClient.class)
           .toProvider(new SupplierProviderBridge<>(masterEnv.getDiscoveryServiceClientSupplier()));
-
-        // TODO: Remove when there is a proper LogAppender to use in K8s
-        bind(LogAppender.class).to(SysOutLogAppender.class).in(Scopes.SINGLETON);
```

Comment:
Might be good to introduce a `protected Module getLogAppenderModule()` that by default return the remote logging append, and have the LogServiceMain override it.